packages/core/src/components/CopyTextButton/CopyTextButton.tsx

```diff
@@ -90,10 +103,4 @@ const CopyTextButton = ({
   );
 };
 
-CopyTextButton.propTypes = {
```

Comment:
Actually keep these around, I think we might want to have prop types for core components since they'll be used from JS quite a lot too.

Reply:
Okay okay, I will revert this.

> they'll be used from JS quite a lot too.

Do you mean other components written in JS? Or plugins and other modules written in JS? 🙇

Reply:
Anything, although core will be all TS

Reply:
The latter. Some plugin makers won't want to or be able to embrace typescript, but will be making use of shared components like this one. We may not notice if they are private plugins out there, but those users will be helped by prop types.

Reply:
Thanks @freben @Rugvip. Sounds good to me.